ci/datamgr.py

```diff
@@ -501,24 +506,17 @@ def bigquery(data_directory, ignore_missing_dependency, **params):
     except google.api_core.exceptions.Conflict:
         pass  # Skip if already created.
 
-    # Set up main data table.
+    # Set up main data tables.
     data_directory = Path(data_directory)
     functional_alltypes_path = data_directory / 'functional_alltypes.csv'
```

Comment:
Minor thing, but I think it makes more sense that these two lines are moved after `# Load main data table.`. It's misleading that they are defined in this section, when they aren't used until later.

Reply:
done in 04b773a40b5317b46b0a8aed939e488ac9f05213